node/forest_libp2p/src/behaviour.rs

```diff
@@ -5,12 +5,13 @@ use ahash::{HashMap, HashSet};
 use forest_encoding::blake2b_256;
 use forest_libp2p_bitswap::BitswapBehaviour;
 use libp2p::{
-    core::{identity::Keypair, PeerId},
+    core::identity::Keypair,
     gossipsub::{
         self, IdentTopic as Topic, MessageAuthenticity, MessageId, PublishError, SubscriptionError,
         ValidationMode,
     },
     identify,
+    identity::PeerId,
```

Comment:
```suggestion
    identity::{self, PeerId, Keypair},
```

Reply:
added

Reply:
Pretty sure you can do it like I showed.

Reply:
(except for the `identify` module. i missed that.)